assets/launcher/lang/sklmessages_pt_BR.properties

```diff
@@ -127,30 +127,30 @@ manager.editor.option.compatibility.name=Modo de compatibilidade
+manager.editor.option.force.info=Isto pode adicionar ou corrigir o suporte de skins para algumas versões (Experimental)
 manager.editor.option.visibility.name=Visibilidade do Launcher
 manager.editor.option.visibility.hide=Esconder o launcher e reabrir quando o jogo fechar
 manager.editor.option.visibility.close=Fechar o launcher quando o jogo iniciar
 manager.editor.option.visibility.keep=Manter o launcher aberto e mostrar a saída de jogo
-manager.editor.option.visibility.keep.info=Esta opção não é recomendada para computadores de baixo custo
+manager.editor.option.visibility.keep.info=Esta opção não é recomendada para computadores de entrada
 manager.editor.option.jvmpath.name=Executável Java
 manager.editor.option.jvmpath.placeholder=Usar o Java Runtime integrado
 manager.editor.option.jvmargs.name=Argumentos da JVM
 manager.editor.option.jvmargs.info=Esta opção ignora os argumentos -Xmx, use memória máxima em vez disso
 
 # New Alerts
-alert.duplicate.window.title=Duplicate Instance Warning
-alert.duplicate.label.title=You already have an instance of Minecraft running.
-alert.duplicate.label.body=If you launch another one in the same directory, they may clash and corrupt your saves. \nThis could cause many issues, in singleplayer or otherwise. We will not be responsible for anything that goes wrong. \nDo you want to start another instance of Minecraft, despite this? \nYou may solve this issue by launching the game in a different directory (see the "Edit Profile" button).
-
-alert.downgrade.window.title=Incompatible Profiles Warning
-alert.downgrade.label.title=Downgrade profiles?
-alert.downgrade.label.body=It looks like you've used a different launcher than this one! \nIf you go back to using this one, we will need to reset your configuration.
-alert.downgrade.button.reset=I'm sure. Reset my settings
-alert.downgrade.button.cancel=Nevermind, close this launcher
-
-alert.invalid.window.title=Invalid Versions Warning
-alert.invalid.label.title=Invalid versions detected :(
-alert.invalid.label.body=The following versions are invalid and cannot be loaded. \nNote: We don't support some versions downloaded through other launchers.
-alert.invalid.button.delete=Delete invalid versions
-alert.invalid.button.close=Close
\ No newline at end of file
+alert.duplicate.window.title=Aviso de Instância Duplicada
+alert.duplicate.label.title=Você já tem uma instância de Minecraft rodando.
+alert.duplicate.label.body=Se você iniciar outra no mesmo diretório, elas podem entrar em conflito e corromper seus dados. \nIsto pode causar muitos problemas, em singleplayer ou não. Não seremos responsáveis por qualquer coisa que dê errado. \nVocê quer iniciar outra instância de Minecraft mesmo assim? \nVocê pode resolver este problema iniciando o jogo em um diretório diferente (veja o botão "Editar Perfil").
+
+alert.downgrade.window.title=Aviso de Perfis Incompatíveis
+alert.downgrade.label.title=Reduzir perfil?
```

Comment:
Aaaah yes, so I think a good translation would be "Reverter atualização do perfil?" But I don't know if a text of that size would look good in the launcher. I'll include it in the commit, but in case it looks bad, using "Redefinir perfil?" is a good idea, although it loses some of the meaning.